index.html

```diff
@@ -16,21 +20,25 @@
       <button
         class="create-task-btn"
         type="button"
-        onclick="location.href='src/task/index.html'"
+        onclick="location.href='task/index.html'"
       >
         Create Tasks
       </button>
       <button
         class="profile-task-btn"
         type="button"
-        onclick="location.href='src/profile/index.html'"
+        onclick="location.href='profile/index.html'"
       >
         Profile
       </button>
       <a
-        href="src/online-members/online-members.html"
-        class="online-members-link"
+        id="user-management-link"
+        class="element-display-remove"
+        href="user-management/index.html"
       >
+        User Management
+      </a>
```

Comment:
why user management link is added in this PR?

Reply:
I did'nt add anything as there is 2 two index.html one in src folder and other one in main folder so as my task is to remove src folder to i'll remove 1 index.html but content is same in both except user management